src/main/java/org/folio/linked/data/mapper/dto/monograph/work/WorkReferenceMapperUnit.java

```diff
@@ -1,45 +1,55 @@
 package org.folio.linked.data.mapper.dto.monograph.work;
 
 import static java.util.Objects.nonNull;
+import static java.util.Optional.ofNullable;
 import static org.folio.ld.dictionary.PredicateDictionary.INSTANTIATES;
+import static org.folio.ld.dictionary.PropertyDictionary.BIBLIOGRAPHY_NOTE;
+import static org.folio.ld.dictionary.PropertyDictionary.LANGUAGE_NOTE;
+import static org.folio.ld.dictionary.PropertyDictionary.NOTE;
 import static org.folio.ld.dictionary.ResourceTypeDictionary.WORK;
 
 import java.util.Collections;
 import java.util.Set;
 import lombok.RequiredArgsConstructor;
+import org.folio.ld.dictionary.PropertyDictionary;
 import org.folio.linked.data.domain.dto.Instance;
-import org.folio.linked.data.domain.dto.WorkReference;
+import org.folio.linked.data.domain.dto.Work;
 import org.folio.linked.data.exception.NotFoundException;
 import org.folio.linked.data.exception.ValidationException;
 import org.folio.linked.data.mapper.dto.common.CoreMapper;
 import org.folio.linked.data.mapper.dto.common.MapperUnit;
 import org.folio.linked.data.mapper.dto.common.SingleResourceMapperUnit;
+import org.folio.linked.data.mapper.dto.monograph.common.NoteMapper;
 import org.folio.linked.data.model.entity.Resource;
 import org.folio.linked.data.repo.ResourceRepository;
 import org.springframework.stereotype.Component;
 
 @Component
 @RequiredArgsConstructor
-@MapperUnit(type = WORK, predicate = INSTANTIATES, dtoClass = WorkReference.class)
+@MapperUnit(type = WORK, predicate = INSTANTIATES, dtoClass = Work.class)
 public class WorkReferenceMapperUnit implements SingleResourceMapperUnit {
```

Comment:
Do we need to rename this class?

Reply:
yes, we do

Reply:
but Instance's field name is still the "_workReference", so we should keep it as is I think

Reply:
sorry, I've misunderstood the question initially